plugins/crashes/frontend/public/javascripts/countly.views.js

```diff
@@ -1835,7 +1835,7 @@ window.CrashgroupView = countlyView.extend({
         // `d` is the original data object for the row
         var str = '';
         if (data) {
-            str += '<div class="datatablesubrow">' +
+            str += '<div id="crash-data-subrow" class="datatablesubrow">' +
```

Comment:
There can be multiple subrows, so it is not a good idea to use `id` there, as id should identify single unique element.

Instead you could just add the id of whole table or of whole crash plugin view